packages/ui/src/components/organisms/ViewAllEditList/LineComponent.tsx

```diff
@@ -18,19 +18,25 @@
 
 import React from 'react';
 import {StyleSheet, View} from 'react-native';
-import {Icon, Text, useThemeColor} from '@axelor/aos-mobile-ui';
+import {Icon, Text} from '../../atoms';
+import {useThemeColor} from '../../../theme';
+import {Line} from './ViewAllEditList';
+
+interface LineComponentProps {
+  line: Line;
+  isSelected: boolean;
+  lines: Line[];
+  setLines: (lines: Line[]) => void;
```

Comment:
two solutions here: 
Use the right type on setLines : 
```
type SetterFunction = (value: Lines[] | ((_current: Lines[]) => Lines[])) => void;
```
Or replace this with handleDelete: (id: number) => void